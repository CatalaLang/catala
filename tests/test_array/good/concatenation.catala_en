## Unit testing for collection concatenation.

```catala
declaration scope A:
   context x content collection integer
   context y content collection integer

scope A:
   definition x equals [0; 1; 2] ++ [3; 4; 5; 6]
   definition y equals x ++ ([7; 8; 9] ++ [10])
```
