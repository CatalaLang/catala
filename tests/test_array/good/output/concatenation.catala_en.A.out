[RESULT] Computation successful! Results:
[RESULT] x = [0; 1; 2; 3; 4; 5; 6]
[RESULT] y = [0; 1; 2; 3; 4; 5; 6; 7; 8; 9; 10]
